src/cheri_insts.sail

```diff
@@ -850,46 +850,24 @@ union clause ast = CSeal : (regidx, regidx, regidx)
 /*!
  * Capability register *cd* is replaced with capability register *cs1*, and is
  * sealed with **otype** equal to the **address** field of capability register
- * *cs2*.
- *
- * ## Exceptions
- *
- * An exception is raised if:
- *   - *cs2*.**tag** is not set.
- *   - *cs2*.**sealed** is set.
- *   - *cs2*.**perms** does not grant **Permit_Seal**.
- *   - *cs2*.**address** $\lt$ *cs2*.**base**.
- *   - *cs2*.**address** $\ge$ *cs2*.**top**.
- *   - *cs2*.**address** $\gt$ [cap_max_otype].
+ * *cs2*. If *cs2* is unable to authorize the sealing, the **tag** field of
+ * *cd* is cleared.
```

Comment:
I suppose the justification here is by symmetry with `CUnseal` (below) and `CCSeal` (which intrinsically has kind of unusual pass-through behavior)... probably worth a slightly expanded comment?